content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -141,8 +141,8 @@ also terminology here is old (s/JNLP/inbound/g), and WebSocket should be mention
    firewalled network connecting to a controller located outside the firewall.
 
  . As a service: First you'll need to manually launch the agent using the above
-   method. After manually launching it, _jenkins-slave.exe_ and
-   _jenkins-slave.xml_ will be created in the agent's work directory. Now go to
+   method. After manually launching it, _jenkins-agent.exe_ and
```

Comment:
Actually the whole previous paragraph is obsolete as of https://github.com/jenkinsci/jenkins/pull/6543 so even if https://github.com/jenkinsci/windows-slave-installer-module/pull/4 had renamed the exe file—which it did not—it would no longer matter.

You can still set up a Windows service for agents, I suppose, but instructions would be different now. You would need to find a winsw expert for details.

Reply:
Hi @jglick thanks for the insight, I was not aware of that prior. Would you happen to know if there is a winsw expert at CloudBees that might be able to help me with this in that case?

Reply:
Not sure offhand. I think @oleg-nenashev used to maintain this; not sure if he is currently available for advice.